lib/objects/account_devices.dart

```diff
@@ -22,12 +22,14 @@ import 'basic_device.dart';
 class AccountDevices {
   String _etag;
   String _kind;
-  String _nextPage;
   List<BasicDevice> _chromeosdevices;
 
+  /// Token used to access the next page of this result.
+  String nextPage;
```

Comment:
Changed to public this variable since it will be modified. I followed https://dart-lang.github.io/linter/lints/unnecessary_getters_setters.html.

Reply:
nit: It's usually preferred to use getter/setters instead of making variables public.